### PR TITLE
Adding stack package folders path for cleanup

### PR DESCRIPTION
**Pull request by bganapa**
## Description
Stack modules were having unwanted dlls, adding  the removal of them in the existing clean up script.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: tools/CleanupBuild.ps1 [hunk 1]
```diff
@@ -7,71 +7,69 @@ Param
...
+    foreach ($RMFolder in $resourceManagerFolders)
     {
-        Write-Verbose "Removing scripts and psd1 in $($stackFolder.FullName)"
-        if (Test-Path -Path "$($stackFolder.FullName)\StartupScripts")
+        $psd1 = Get-ChildItem -Path $RMFolder.FullName -Filter "$($RMFolder.Name).psd1"
+        Import-LocalizedData -BindingVariable ModuleMetadata -BaseDirectory $psd1.DirectoryName -FileName $psd1.Name
+        
+        $acceptedDlls = @()
+        $acceptedDlls += $ModuleMetadata.NestedModules
+        $acceptedDlls += $ModuleMetadata.RequiredAssemblies
```
Comment (deathly809) [inline tools/CleanupBuild.ps1]: We don't fill this in, we need to do a PR to add these.

**Comment by bganapa** [comment]
successful ps-sign https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1219/

**Comment by bganapa** [comment]
Tests are broken. I have messed up something.. :(

**Comment by bganapa** [comment]
@deathly809  Did you publish your review? I dont see the comments

**Comment by bganapa** [comment]
@deathly809  I see it, I have added the requred assemblies

File: tools/CleanupBuild.ps1 [hunk 2]
```diff
@@ -2,76 +2,82 @@
...
-$webdependencies = @("Microsoft.Web.Hosting.dll", "Microsoft.Web.Delegation.dll", "Microsoft.Web.Administration.dll", "Microsoft.Web.Deployment.Tracing.dll")
-Get-ChildItem -Include $webdependencies -Recurse -Path $output | Remove-Item -Force
+    Write-Verbose "Removing unneeded web deployment dependencies"
+    $webdependencies = @("Microsoft.Web.Hosting.dll", "Microsoft.Web.Delegation.dll", "Microsoft.Web.Administration.dll", "Microsoft.Web.Deployment.Tracing.dll")
+    Get-ChildItem -Include $webdependencies -Recurse -Path $path | Remove-Item -Force
+}
 
-$resourceManagerFolders = Get-ChildItem -Path $resourceManagerPath -Directory
-foreach ($RMFolder in $resourceManagerFolders)
+$resourceManagerPaths = @($resourceManagerPath)
```
Comment (deathly809) [inline tools/CleanupBuild.ps1]: You should move this above with the other if(..) statement so you can do it in one go.

Comment (deathly809) [reply tools/CleanupBuild.ps1]: Nevermind, it could be possible, but very rare that one folder exists but not the other.

**Comment by markcowl** [comment]
sign build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1227/
netsign build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/180/

**Comment by bganapa** [comment]
ps-netsign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/181/console
ps -sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1228/

**Comment by bganapa** [comment]
Rikicked ps-sign after Compute PR got merged, 
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1229/console
psnetsign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/

**Comment by maddieclayton** [comment]
Checked over dlls in ResourceManager and they look correct.  Passing Travis job is here: https://travis-ci.org/Azure/azure-powershell/builds/426798935, merging now.

**Pull request merged**